examples/todos/src/routes.rs

```diff
@@ -234,7 +226,10 @@ async fn add_user_extension(
                 .get_user(&session.user_id)
                 .await
                 .expect("Failed to get user");
-            request.extensions_mut().insert(user);
+            request.extensions_mut().insert(user.clone());
```

Comment:
Could you add a comment here why we insert the user twice, once as an Option<User> and once as a User.

Reply:
Done! I'm sure there are more elegant ways but I don't know the extension api well.